cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -177,7 +177,10 @@ data:
         forward . /etc/resolv.conf {
            max_concurrent 1000
         }
-        cache 30
+        cache 30 {
+           disable success {{ .DNSDomain }}
+           disable denial {{ .DNSDomain }}
```

Comment:
no other users have ever reported the need for these params. so based on that it seems setup specific to me.

@chrisohaver could you please review this change to determine if this is something to add by default in kubeadm? related to:
- https://github.com/kubernetes/kubernetes/issues/128325